Documentation/platforms/arm/stm32f4/boards/stm32f401rc-rs485/index.rst

```diff
@@ -632,3 +632,35 @@ Now you will be able to access the Device Configuration over Telnet::
        help, ?   - show help
        reset     - reset the board
        exit      - exit shell
+
+max7219
+-------
+
+Configures the NuttShell (nsh) over USB Serial (check usbserial configuration) and enables LCD driver with
+MAX7219 for 8x8 LED matrix::
```

Comment:
Include information with the nxhello file/variable you need to modify to print only a letter in the display, explain that user needs to do it because the 8x8 pixels can only display a single letter